projects/core/src/model/misc.model.ts

```diff
@@ -122,4 +122,10 @@ export interface BaseSite {
   urlEncodingAttributes?: string[];
   baseStore?: BaseStore;
   requiresAuthentication?: boolean;
+  captchaConfig?: CaptchaConfig;
+}
+
+export interface CaptchaConfig {
+  enabled: boolean;
+  publicKey?: string;
```

Comment:
Can we leave a helpful jsdoc comment to explain a bit more about this config? eg. Public key is provided by the backend...